### ISSUE TITLE
EMBL Colours and colour profiles

### ISSUE THREAD

**Issue by khawkins98** [embl-communications/design-system#20]
The Design team has noticed issues with the display of colours being too bright in several situations. 

## TL;DR

We should use Illustrator to bring colours back "in gamut". I have some revised colours to test below.

## Example scenarios

- PPT slides are too saturated when shown
- Large green areas on web pages 

The common issue is colour management, specifically when a program or OS is not applying any colour management.

## The issue

- The EMBL green is `7482-C`: `RGB 0,159,77`
- When a program does apply colour management of the OS this is converted to a display value  along the lines of `rgb (71, 158, 91)`
- Programmes like FireFox and PowerPoint will show the raw value, resulting in a much brighter green

### Evidence

Firefox: 
![image](https://user-images.githubusercontent.com/928100/66749682-ca649e00-ee8a-11e9-9c1f-b67c882f7124.png)

Chrome:
![image](https://user-images.githubusercontent.com/928100/66749695-cf295200-ee8a-11e9-9830-07d69d67e323.png)

Further evidence: 

However if I set my display to use sRBG (instead of the manufacturer calibration) both show the same raw (oversaturated) green:

![image](https://user-images.githubusercontent.com/928100/66749832-229ba000-ee8b-11e9-84ae-99662294e599.png)

### Deeper notes

We're getting this problem as we're seeing two issues coincide:

1. More devices are capable of showing the wider-gamut Adobe RGB spectrum but the software was developed expecting a limitation closer to sRGB; example:

![image](https://user-images.githubusercontent.com/928100/66753309-c2f5c280-ee93-11e9-974f-b1783936d8f0.png)
[source](https://www.eizo.be/en/knowledge/monitor-expertise/understanding-color-gamut/)

2. The sRGB gamut is more limited in the green spectrum 

![image](https://user-images.githubusercontent.com/928100/66753402-02bcaa00-ee94-11e9-9218-1fc1c6db1f32.png)
 
## Solutions

As colour of everyone's device is not an option, we can consider a green that is more friendly to sRGB and native RGB screens.

- Option A: Instead of [Pantone 7482 C](https://www.pantone.com/color-finder/7482-C) `rgb (0, 159, 77)`, we could use [Pantone 7482 CP](https://www.pantone.com/color-finder/7482-CP) `RGB rgb(0, 161, 83)`
    - This does not result in a substantial correction towards the Chrome's colour managed `71, 158, 91` 
- Option B: Use Adobe Illustrator's tool to bring colours back in gamut (consistently and while using the sRGB space)
    - Results in rgb(24,151,76). Which helps bring down the oversaturation without muddying too much in other areas. It's closer to a "safe" spectrum
 
Here's what "B" looks like: Firefox on left, Chrome right

![image](https://user-images.githubusercontent.com/928100/66752494-cdaf5800-ee91-11e9-9b70-5c898dc9f4f5.png)

## Next steps

Here is the EMBL green colour in gamut for sRGB. We should it PPT, web, and other places:

- EMBL Green
    - current: `rgb(0, 159, 77)`
    - in gamut: `rgb(24,151,76)`
- EMBL Green, Darkest Tint
    - current: `rgb(10, 80, 50)`
    - in gamut: can stay as is
- EMBL Green, Dark Tint
    - current: `rgb(0, 123, 83)`
    - in gamut: can stay as is
- EMBL Green, Light Tint
    - current: `rgb(108, 194, 74)`
    - in gamut: can stay as is
- EMBL Green, Lightest Tint
    - current: `rgb(204, 226, 193)`
    - in gamut: can stay as is

When we look at the colour gamut, it isn't that surprising that only the EMBL Green would be out of gamut: the more +green +blue -red a colour gets, the worse the colour coverage is (EMBL Blue is also out of gamut for this reason). 

Of course, saying a colour is "in gamut" doesn't mean it won't have issues, but we'll see fewer issues when dealing with the sRGB colour space rendered on higher-gamut devices — a good analogy for what we're seeing in Firefox and Powerpoint would be stretching an image aspects ratio to fill a wider space, in this case a wider gamut.

For quick testing: you can open this page on various devices, browsers https://dev.beta.embl.org/guidelines/design/brand-guidelines/colour/

**Comment by khawkins98** [comment]
## Other colours out of gamut

Went through and tested the other colours; here's what else was out of gamut:

- EMBL Red
    - current: `rgb(228, 0, 70)`
    - in gamut: `rgb(212, 22, 69)`
- EMBL Blue
    - current: `rgb(49, 122, 226)`
    - in gamut: `rgb(59, 111, 182)`
- EMBL Blue, Dark tint
    - current: `rgb(0, 61, 165)`
    - in gamut: `rgb(25, 63, 144)`
- EMBL Purple
    - current: `rgb(130, 70, 175)`
    - in gamut: `rgb(115, 69, 149)`
- EMBL Orange
    - current: `rgb(255, 163, 0)`
    - in gamut: `rgb(244, 158, 23)`
- EMBL Orange, dark tint
    - current: `rgb(190, 84, 0)`
    - in gamut: `rgb(182, 84, 23)`
- EMBL Yellow
    - current: `rgb(255, 205, 0)`
    - in gamut: `rgb(244, 198, 31)`
- EMBL Bright Green
    - current: `rgb(168, 199, 0)`
    - in gamut: `rgb(161, 190, 31)`
- EMBL Bright Green, dark tint
    - current: `rgb(132, 189, 0)`
    - in gamut: `rgb(127, 180, 40)`

If it's not listed, should be fine (according to the below method).

This isn't to say these are the RGBs we should use, but if we are seeing issues with a colour these values are "safer". These are just suggestions to consider … really I don’t think any of our colours are _wrong_. Rather the above colours might go weird when software doesn’t respect colour management. … Sometimes maybe we don’t need to care if it’s out of gamut: EMBL Yellow is going to be retina scaring whether or not we bring it down, but given how it’s likely to be used that’s probably OK.

## Method

- RGB document in illustrator
- Colour settings (keyboard shortcut: Shift-Command-K)
    - Working space/profile: RGB: sRGB ICE616966-2.1
    - Management policy: RGB: convert to working space
    - Conversion options:
       - Engine: Adobe (ACE)
       - Intent: Perceptual

**Comment by khawkins98** [comment]
The colours were updated using this approach in the above PR. 

As a final task we should document our methodology on this.
